noir_stdlib/src/collections/bounded_vec.nr

```diff
@@ -7,7 +7,7 @@ struct BoundedVec<T, let MaxLen: u32> {
 
 impl<T, let MaxLen: u32> BoundedVec<T, MaxLen> {
     pub fn new() -> Self {
-        let zeroed = crate::unsafe::zeroed();
+        let zeroed = crate::unsafe_func::zeroed();
```

Comment:
I don't think naming the module `unsafe_func` reads well. I don't have a great renaming suggestion either though. I've heard `trustme` used in place of `unsafe`  before but it reads as less professional.

Reply:
I think this was done by Tom. I don't have a preference for this name. Which one should we use?

Reply:
Maybe `mem` like Rust's memory module? We would lose the unsafeness of the function unfortunately.